content/departments/ce-support/support/process/support-time-off-guidelines.md

```diff
@@ -0,0 +1,29 @@
+# Support Time-Off Guidelines
+
+The handbook provides general [guidelines](../../../../benefits-pay-perks/benefits-perks/time-off/submitting-time-off.md) regarding our unlimited PTO policy for both planned and unplanned time off. It essentially states: “At Sourcegraph, every teammate has agency over how they create a balance between work and rest, so that we can stay productive and innovative over the long term.” Due to both the customer-facing aspect of our work and the unplannable nature of support work, it is important that PTO is being taken in a way that is healthy for the team and our customers while still honoring the intention of this policy.
+
+As such, to amend the company-wide guidelines for PTO, the following additional guidance is required when taking time off moving forward:
+
+- **Communication**: It is important that planned [time off is submitted](https://handbook.sourcegraph.com/benefits-pay-perks/benefits-perks/time-off/submitting-time-off/#recording-paid-time-off-via-roots) early enough for your manager to assess its impact to the team and mitigate it. As best practice, when planning multiple days off, at least **one week** of advanced notice and planning is desired. Even if you know in advance you are taking a single day off, it’s still desired to give your teammates as much advance notice as possible. With unplanned time off, communication (with your manager and the team) is even more important especially if it impacts your ability to deliver on your commitments.
+
+- **Frequency**: While we encourage folks to use their discretion in taking time off, we also implore that good judgment is exercised in doing so. Ensuring that time off is scheduled in a way that does not adversely impact the wider team and our customers exemplifies our “high agency”, “work as a team” and “customer-first” values. A few practices that do not would be:
+
+  1.           Requesting extended time off last minute which inhibits the ability to properly plan for coverage.
+  1.           Regularly taking time off on Mondays or Fridays to achieve a ‘long weekend’ or a ‘4-day workweek’.
```

Comment:
I think you were trying to iterate her, so the second 1. should be 2.
Every other thing LGTM

Reply:
On markdown numbering is on auto so it should be good on the doc.